domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -200,7 +203,8 @@ class StateRetriever @Inject constructor(
         .setIsNegative(correctAnswerObject.getBoolean("isNegative"))
         .build()
     } else {
-      CorrectAnswer.newBuilder().setCorrectAnswer(containerObject.getString("correct_answer")).build()
+      CorrectAnswer.newBuilder().setCorrectAnswer(containerObject.getString("correct_answer"))
```

Comment:
This wrapping doesn't seem quite right. We should wrap at the higher syntactical element, which probably is the outer ``.`` in this case. That means we should break at each one:

```kotlin
CorrectAnswer.newBuilder()
  .setCorrectAnswer(containerObject.getString("correct_answer"))
  .build()
```

Ditto elsewhere.